src/global/signal_handler.h

```diff
@@ -20,12 +20,12 @@
 
 typedef void (*signal_handler_t)(int);
 
-#ifdef HAVE_SIGDESCR_NP
+#if defined HAVE_SIGDESCR_NP
 # define sig_str(signum) sigdescr_np(signum)
-#elif HAVE_REENTRANT_STRSIGNAL
-# define sig_str(signum) strsignal(signum)
-#else
+#elif !defined HAVE_REENTRANT_STRSIGNAL
```

Comment:
instead of using `!defined` i'd suggest just put
```c++
#elif defined(HAVE_REENTRANT_STRSIGNAL)
#  define sig_str(signum) strsignal(signum)
#else
#  define sig_str(signum) sys_siglist[signum]
#endif
```

Reply:
Ok, that works as well.  Will amend.